jenkins/opensearch/Jenkinsfile

```diff
@@ -3,6 +3,7 @@ pipeline {
     triggers {
       parameterizedCron '''
         H */2 * * * %INPUT_MANIFEST=1.2.0/opensearch-1.2.0.yml
+        0 1 * * * %INPUT_MANIFEST=2.0.0/opensearch-2.0.0.yml
```

Comment:
It is not recommended to run workflows at the exact time in Jenkins.
Suggest H 1 * * * if you want to run the task between 1 and 2 every day for 2.0.0.

Reply:
If you want to run 1:00 to 1:10 use H(0-10) * * * *.

Reply:
Updated.